bindings/ios/MEGASdk.mm

```diff
@@ -2161,6 +2161,11 @@ - (SMSState)smsAllowedState {
     return (SMSState)self.megaApi->smsAllowedState();
 }
 
+- (NSString *)smsVerifiedPhoneNumber {
+    char *number = self.megaApi->smsVerifiedPhoneNumber();
```

Comment:
Memory leak here.
smsVerifiedPhoneNumber  -> You take the ownership of the returned value.
We should release the memory.

Reply:
It's handled now. Please have a look